providers/google-workspace/resources/google-workspace.lr

```diff
@@ -175,6 +175,7 @@ private googleworkspace.role @defaults("name") {
 // Google Workspace apps reports
 private googleworkspace.report.apps {
   drive() []googleworkspace.report.activity
```

Comment:
Both of these need a description so they get documented properly

Reply:
Thanks @tas50 , noted.